### PR TITLE
added inCode flag

### PR DESCRIPTION
**Pull request by bobzel**
added inCode flag to inputRule specs to allow inputRules to selectively apply to code blocks.

### CODE REVIEW

File: src/inputrules.ts [hunk 1]
```diff
@@ -35,12 +36,17 @@ export class InputRule {
       /// When set to false,
       /// [`undoInputRule`](#inputrules.undoInputRule) doesn't work on
       /// this rule.
-      undoable?: boolean
+      undoable?: 'boolean',
```
Comment (marijnh) [inline src/inputrules.ts]: I don't think this should be a string.

File: src/inputrules.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ export class InputRule {
 
   /// @internal
   undoable: boolean
+  inCode?: 'allowed' | 'only';
```
Comment (marijnh) [inline src/inputrules.ts]: Could you make this `boolean | "only"`, so people can also explicitly provide `false`?

File: src/inputrules.ts (hunk 1 above)
Comment (bobzel) [reply src/inputrules.ts]: yes, you're right, don't know how I changed that.

File: src/inputrules.ts (hunk 2 above)
Comment (bobzel) [reply src/inputrules.ts]: done.

**Comment by bobzel** [comment]
Just curious since this is my first pull request -- how long should I expect it to take to have it merged?  days, weeks, months?
Thanks and sorry for being so naive to this process.

**Comment by marijnh** [comment]
Merged as 66d47eaa489ff4eab, tidied up in a993c59c111dc5973102f55. Released as part of prosemirror-inputrules 1.4.0

**Pull request closed without merging**